### ISSUE TITLE
the loss formula 

### ISSUE THREAD

**Issue by yangc20** [eriklindernoren/PyTorch-YOLOv3#549]
I use the loss formula in models.py ,loss_conf =1* loss_conf_obj + 100* loss_conf_noobj.After training 2000 images for 20 epoch with pretrain weight,the total loss is about 0.9,and I test my image ,there are many boxes in each image,and the most classes of the boxes  are wrong.(I have changed the conf_thre and the nms_thre but the error still exists).Anyone else know，please?

**Comment by Flova** [comment]
Is this issue still relevant/occurring?

**Comment by Flova** [comment]
I closed this issue due to inactivity. Feel free to reopen for further discussion.
